modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts

```diff
@@ -14,6 +14,7 @@
 
 import {Parameters, parametersFormater} from '.';
 import {axios} from './liferay/api';
+const nowDate = new Date().toISOString().split('T')[0];
```

Comment:
I believe we already have a current date variable on _dateFormatter_ folder. Try to find it.

Reply:
Done